src/Client/Updater.php

```diff
@@ -375,8 +318,6 @@ protected function verify(string $target, StreamInterface $data): void
         if ($targetsMetadata === null) {
             throw new NotFoundException($target, 'Target');
         }
-        $maxBytes = $targetsMetadata->getLength($target) ?? static::MAXIMUM_DOWNLOAD_BYTES;
```

Comment:
There's no further need for this, because SizeCheckingLoader takes care of it before the stream even reaches this method.